Documents/API.md

```diff
@@ -287,6 +291,86 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
     }
     ```
 
+    Sorted by Row:
+    ```json
+    {
+      "rows": [
+        {
+          "rowId": "Transmittal Sheet",
+          "edits": [
+            {
+              "editId": "S020",
+              "description": "Agency code must = 1, 2, 3, 5, 7, 9. The agency that submits the data must be the same as the reported agency code.",
+              "fields": {
+                "Agency Code": 4
+              }
+            }
+          ]
+        },
+        {
+          "rowId": "8299422144",
+          "edits": [
+            {
+              "editId": "S020",
+              "description": "Agency code must = 1, 2, 3, 5, 7, 9. The agency that submits the data must be the same as the reported agency code.",
+              "fields": {
+                "Agency Code": 11
+              }
+            }
+          ]
+        },
+        {
+          "rowId": "4977566612",
+          "edits": [
+            {
+              "editId": "V550",
+              "description": "Lien status must = 1, 2, 3, or 4.",
+              "fields": {
+                "Lien Status": 0
+              }
+            },
+            {
+              "editId": "V555",
+              "description": "If loan purpose = 1 or 3, then lien status must = 1, 2, or 4.",
+              "fields": {
+                "Loan Purpose": 1,
+                "Lien Status": 0
+              }
+            },
+            {
+              "editId": "V560",
+              "description": "If action taken type = 1-5, 7 or 8, then lien status must = 1, 2, or 3.",
+              "fields": {
+                "Action Taken Type": 3,
+                "Lien Status": 0
+              }
+            }
+          ]
+        }
+      ],
+      "macroResults": {
```

Comment:
Good question.  `macro` is a keyword in Scala, so it's hard to name things just that. I'll try to find out what it takes